machines/2.0/config_compilers.xml

```diff
@@ -0,0 +1,1224 @@
+  <LAPACK_LIBDIR> /glade/apps/opt/lib </LAPACK_LIBDIR>
+  <SCC MPILIB="mpi-serial">gcc</SCC>
+  <SFC MPILIB="mpi-serial">gfortran</SFC>
+</compiler>
+
+<compiler MACH="yellowstone" COMPILER="intel">
+  <CFLAGS>
+    <append MPILIB="mpich2">  -xHost </append>
+  </CFLAGS>
+  <CPPDEFS>
+    <append MPILIB="mpich2"> -DINTEL_MKL -DHAVE_SSE2 </append>
+  </CPPDEFS>
+  <FFLAGS>
+    <append MPILIB="mpich2">  -xHost </append>
+  </FFLAGS>
+  <MPICXX MPILIB="mpich2">mpiicpc</MPICXX>
+  <PAPI_INC MPILIB="mpich2"> /glade/apps/opt/papi/5.3.0/intel/12.1.5/include/</PAPI_INC>
+  <PAPI_LIB MPILIB="mpich2">/glade/apps/opt/papi/5.3.0/intel/12.1.5/lib64 </PAPI_LIB>
+  <PFUNIT_PATH MPILIB="mpi-serial" compile_threaded="false">$ENV{CESMDATAROOT}/tools/pFUnit/pFUnit3.1_yellowstone_Intel15.0.1_noMPI_noOpenMP</PFUNIT_PATH>
+  <PFUNIT_PATH MPILIB="mpich2" compile_threaded="true">$ENV{CESMDATAROOT}/tools/pFUnit/pFUnit3.1_Intel15.0.1_MPI</PFUNIT_PATH>
+  <!-- Needed due to the way that netcdf is loaded on yellowstone -->
+  <SCC MPILIB="mpi-serial">icc</SCC>
+  <SFC MPILIB="mpi-serial">ifort</SFC>
+  <SCC MPILIB="mpich2">${MPICC}</SCC>
+  <SFC MPILIB="mpich2">${MPIFC}</SFC>
+  <SLIBS>
+    <append MPILIB="mpich2"> -Wl,-rpath ${PAPI_LIB} -L${PAPI_LIB} -lpapi</append>
+  </SLIBS>
+  <TRILINOS_PATH MPILIB="mpich2">$ENV{TRILINOS_PATH}</TRILINOS_PATH>
+</compiler>
+
+<compiler MACH="yellowstone" COMPILER="pgi">
+  <CFLAGS>
+    <append DEBUG="FALSE"> -O </append>
+  </CFLAGS>
+  <FFLAGS>
+    <append DEBUG="FALSE"> -O </append>
+  </FFLAGS>
+  <SLIBS>
+    <append> -llapack -lblas </append>
+  </SLIBS>
+  <SCC MPILIB="mpi-serial">pgcc</SCC>
+  <SFC MPILIB="mpi-serial">pgfortran</SFC>
+</compiler>
+
+<compiler COMPILER="intel" MACH="lbl-lr3">
+  <CPPDEFS>
+    <append MODEL="gptl"> -DHAVE_VPRINTF -DHAVE_TIMES -DHAVE_GETTIMEOFDAY </append>
+  </CPPDEFS>
+  <SLIBS>
+    <append> -lnetcdff -lnetcdf -mkl </append>
+  </SLIBS>
+  <FFLAGS>
+    <append DEBUG="TRUE"> -ftrapuv </append>
+  </FFLAGS>
+  <CFLAGS>
+    <append DEBUG="TRUE"> -ftrapuv </append>
+  </CFLAGS>
+  <NETCDF_PATH>$ENV{NETCDF_DIR}</NETCDF_PATH>
+</compiler>
+
+<compiler COMPILER="intel" MACH="lbl-lr2">
+  <CPPDEFS>
+    <append MODEL="gptl"> -DHAVE_VPRINTF -DHAVE_TIMES -DHAVE_GETTIMEOFDAY </append>
+  </CPPDEFS>
+  <SLIBS>
+    <append> -lnetcdff -lnetcdf -mkl </append>
+  </SLIBS>
+  <FFLAGS>
+    <append DEBUG="TRUE"> -ftrapuv </append>
+  </FFLAGS>
+  <CFLAGS>
+    <append DEBUG="TRUE"> -ftrapuv </append>
+  </CFLAGS>
+  <NETCDF_PATH>$ENV{NETCDF_DIR}</NETCDF_PATH>
+</compiler>
+
+<compiler COMPILER="gnu" MACH="modex">
+  <SLIBS>
+    <append> -L$ENV{HDF5_HOME}/lib -lhdf5_fortran -lhdf5 -lhdf5_hl -lhdf5hl_fortran </append>
+    <append> -L$ENV{NETCDF_PATH}/lib/ -lnetcdff -lnetcdf -lcurl -lblas -llapack</append>
+  </SLIBS>
+  <CPPDEFS>
+     <append MODEL="gptl"> -DHAVE_VPRINTF -DHAVE_GETTIMEOFDAY -DHAVE_BACKTRACE </append>
+  </CPPDEFS>
+</compiler>
+
+<compiler MACH="hydra" COMPILER="gnu">
```

Comment:
I know that VUB's climate group also uses other clusters apart from ours. Since they might be copying the source code between clusters, I prefer to not remove any other cluster from the file. Ideally, we should be able to push our setting upstream. I'll investigate how that can be done in the future.